src/modules/account/account.service.ts

```diff
@@ -0,0 +1,159 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { CreateAccountDto } from './dtos/create-account.dto';
+import { generate } from 'generate-password';
+import { UserRoles } from '@prisma/client';
+import { PrismaService } from '../prisma/prisma.service';
+import { UpdateAccountDto } from './dtos/update-account.dto';
+import { ExceptionsHandler } from '@nestjs/core/exceptions/exceptions-handler';
+
+@Injectable()
+export class AccountService {
+  constructor(private prisma: PrismaService) {}
+
+  async createAccount(data: CreateAccountDto, userId: number) {
+    try {
+      if (!userId) return null;
```

Comment:
@Akeron44 If we don't have a user id, something is wrong. We should throw an exception instead